main.go

```diff
@@ -488,6 +524,16 @@ func applyArgs(dryrun bool, file string) []string {
 	return args
 }
 
+// printTrimmedError prints the last line of stderrbuf to stderr
+func printTrimmedError(stderrbuf bytes.Buffer) {
+	var lastLine string
+	scanner := bufio.NewScanner(strings.NewReader(string(stderrbuf.Bytes())))
```

Comment:
`bufio.NewScanner` accepts an `io.Reader`, which is what `*bytes.Buffer` implements.

You can simplify this to `bufio.NewScanner(&stderrbuff)`